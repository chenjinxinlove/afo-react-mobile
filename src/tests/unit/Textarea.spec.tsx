import * as React from 'react';
import * as renderer from 'react-test-renderer';
import DemoTextarea from '../../pageages/textarea/demo';

describe('Button', () => {
  // 测试快照
  it('test snapshots', () => {
    const component = renderer.create(
      <DemoTextarea />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
