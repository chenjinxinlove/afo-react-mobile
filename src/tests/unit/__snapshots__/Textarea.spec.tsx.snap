// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test snapshots 1`] = `
<div>
  <p>
    基础用法
  </p>
  <div
    className="afo-textarea-wrapper"
    style={undefined}
  >
    <textarea
      className="afo-textarea"
      disabled={false}
      maxLength={60}
      onBlur={[Function]}
      onFocus={[Function]}
      onInput={[Function]}
      placeholder="please type here..."
      value="xxxxxvalue"
    />
    <span
      className="afo-textarea__indicator"
    >
      50
    </span>
  </div>
</div>
`;
